examples/README.md

```diff
@@ -1,22 +1,84 @@
-# Contents of this Document
+# Examples showcasing TorchServe Features and Integrations
+
+## TorchServe Internals
+
 * [Creating mar file for an eager mode model](#creating-mar-file-for-eager-mode-model)
 * [Creating mar file for torchscript mode model](#creating-mar-file-for-torchscript-mode-model)
-* [Serving torchvision image classification models](#serving-image-classification-models)
 * [Serving custom model with custom service handler](#serving-custom-model-with-custom-service-handler)
-* [Serving text classification model](#serving-text-classification-model)
-* [Serving text classification model with scriptable tokenizer](#serving-text-classification-model-with-scriptable-tokenizer)
+* [Creating a Workflow](#serving-image-classification-workflow)
+* [Custom Metrics](custom_metrics)
+* [Batch Processing](image_classifier/resnet_152_batch)
```

Comment:
nit - let's call this dynamic batch processing

Reply:
Done